ProcessStub-Vanguard.sln

```diff
@@ -9,19 +9,24 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "RTCV", "RTCV", "{EBBC4835-E
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Libraries", "Libraries", "{2956F1A8-9BE7-4B90-861F-E6A13D0D9260}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CorruptCore", "..\RTCV\Source\Libraries\CorruptCore\CorruptCore.csproj", "{B5633A86-DC66-4723-88EB-7DFC95458072}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CorruptCore", "..\RTCV\Source\Libraries\CorruptCore\CorruptCore.csproj", "{B5633A86-DC66-4723-88EB-7DFC95458072}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NetCore", "..\RTCV\Source\Libraries\NetCore\NetCore.csproj", "{7E106AEC-5614-4206-9E4D-25759C90E8AE}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NetCore", "..\RTCV\Source\Libraries\NetCore\NetCore.csproj", "{7E106AEC-5614-4206-9E4D-25759C90E8AE}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Vanguard", "..\RTCV\Source\Libraries\Vanguard\Vanguard.csproj", "{364C705E-B7AB-4A94-A59B-88899CE65958}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Vanguard", "..\RTCV\Source\Libraries\Vanguard\Vanguard.csproj", "{364C705E-B7AB-4A94-A59B-88899CE65958}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "StandaloneRTC", "..\RTCV\Source\Frontend\StandaloneRTC\StandaloneRTC.csproj", "{597D9202-F18D-4B51-BB89-D7E111BAF8B5}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "StandaloneRTC", "..\RTCV\Source\Frontend\StandaloneRTC\StandaloneRTC.csproj", "{597D9202-F18D-4B51-BB89-D7E111BAF8B5}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UI", "..\RTCV\Source\Frontend\UI\UI.csproj", "{BD376BB2-17EC-4FB0-A9FE-6B1BBF4AF3E3}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "UI", "..\RTCV\Source\Frontend\UI\UI.csproj", "{BD376BB2-17EC-4FB0-A9FE-6B1BBF4AF3E3}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ProcessCorrupt", "ProcessCorrupt\ProcessCorrupt.csproj", "{A1A6429D-F28E-49E2-9B17-2FF83A9170FD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Common", "..\RTCV\Source\Libraries\Common\Common.csproj", "{A31C17E1-2EAB-434F-A022-20193F2CEAB7}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Common", "..\RTCV\Source\Libraries\Common\Common.csproj", "{A31C17E1-2EAB-434F-A022-20193F2CEAB7}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{74D1ADDA-16F0-4DEF-BDFF-16498A243EF3}"
+	ProjectSection(SolutionItems) = preProject
+		.editorconfig = .editorconfig
```

Comment:
This intention of the `ProcessStub-Vanguard.sln` diff is just to add `.editorconfig` as a solution file for Visual Studio. I'm not sure why all of the other project GUIDs got regenerated.

Reply:
A bit odd but shouldn't break anything. I'd rather not fight VS